kubernetes/controller/test/utils/utils.go

```diff
@@ -210,7 +210,7 @@ func CreateNamespace(ctx context.Context, ns string) error {
 
 // Delete Kubernetes namespace.
 func DeleteNamespace(ctx context.Context, ns string) error {
-	cmd := exec.CommandContext(ctx, "kubectl", "delete", "ns", ns, "--ignore-not-found=true", "--cascade=background")
+	cmd := exec.CommandContext(ctx, "kubectl", "delete", "ns", ns, "--ignore-not-found=true", "--cascade=foreground")
```

Comment:
Isn't background the default so you might as well just delete this option all together?

Reply:
Yep: https://kubernetes.io/docs/tasks/administer-cluster/use-cascading-deletion/#use-background-cascading-deletion

Reply:
No wait. You changed this to foreground. :D